highlight.js

```diff
@@ -79,29 +79,6 @@
                 } else {
                     collect(node);
                 }
-            },
-
-            /**
-             * @see https://github.com/angelozerr/tern-lint
-             *
-             * Detects top-level identifiers, e.g. the object in
-             * `object.property` or just `object`.
-             */
-            Identifier: function (node, state) {
-                var type = infer.expressionType({
```

Comment:
We also don't need the `infer` dependency now (it's injected in the beginning of the file).